packages/chatbot-server-mongodb-public/src/evalEnvVars.ts

```diff
@@ -0,0 +1,6 @@
+export const evalEnvVars = {
+  JUDGE_LLM: "",
+  JUDGE_EMBEDDING_MODEL: "",
+  // TODO: replace with Azure OpenAI when supported in Autoevals
```

Comment:
Is there a GH issue or something tracking this?

Reply:
ye https://jira.mongodb.org/secure/RapidBoard.jspa?rapidView=2783&view=detail&selectedIssue=EAI-504&quickFilter=23556#

Reply:
Wow love that Braintrust turnaround time!